src/app/components/navbar/navbar.component.ts

```diff
@@ -79,4 +81,10 @@ export class NavbarComponent implements OnInit, OnDestroy {
   private removeLeadingSlash(s = '') {
     return s.replace(/^\/+/g, '');
   }
+
+  //@doritrieur in order to get the first part of the path and compare to the route content.
+  private getFirstPathToken(s = '') {
```

Comment:
I think that we already have a mechanism of routing and there we know where are we. Please consult with Celine Zittoun for help. 
The problem with this solution is that if someday someone will change the path, the title won't fit...

Reply:
Hi Dorit,

Any update about this?